### DISCUSSION TITLE
How do I increase the player maximum for FFA, Teams, and Coop?

### DISCUSSION THREAD

**Discussion by joshdajish** [efroemling/ballistica#393] [Q&A]
I got this game a while ago on my high school computers. The game files are on a shared cloud network connected to every school computer that is horribly monitored... Either way, I tried following the Bombsquad modding guide 2 but the commands are not working. It says something like "bs" object not found...

**Comment by Delitel-WEB** [comment]
Check the version of the game. If the game version is 1.5+, then starting from this version the API has been changed for the Ballistica engine
https://github.com/efroemling/ballistica/blob/4d31d59467a4cae8140da47812a6e8fa7ae208ba/docs/ba_module.md
https://github.com/efroemling/ballistica/wiki

if the version of the game is lower, then here is the old api:
https://www.froemling.net/docs/bombsquad-python-api

**Reply by joshdajish** [reply]
the version is 1.6.4. I just don't know what commands I need to enter to increase the player maximum for matches.

**Reply by Delitel-WEB** [reply]
I think you need to consider: ba.App:
https://github.com/efroemling/ballistica/blob/4d31d59467a4cae8140da47812a6e8fa7ae208ba/docs/ba_module.md#baapp

**Comment by TheNotEasy** [comment]
try this "`ba.Session.on_player_request = lambda a, b: True`"
Note that removing limit not increasing
